### ISSUE TITLE
Strange behavior with Xcode 8

### ISSUE THREAD

**Issue by vincentisambart** [CocoaPods/Xcodeproj#405]
When using Xcodeproj 1.2.0, if the Xcode version selected with `xcode-select` is Xcode 8 beta 6, the following code displays `KO`. It displays `OK` if you're using Xcodeproj 1.1.0 or if you've selected Xcode 7.
```ruby
require "pathname"
require "xcodeproj"

# The project can be anything, a default iOS project generated by Xcode will do.
xcodeproj_path = Pathname.new("Test.xcodeproj")
xcodeproj = Xcodeproj::Project.open(xcodeproj_path)
xcodeproj.save # If this line is commented out, the problem disappears.

# Creating a directory named "test"
# (the bug also occurs is the directory is not created by the code but beforehand)
test_path = Pathname.new("/tmp/mylittletestdirectory")
test_path.mkpath
current_path = Dir.pwd
# Go to /tmp/mylittletestdirectory.
Dir.chdir(test_path) do
  if current_path != Dir.pwd
    puts "OK"
  else
    puts "KO" # The current directory should have changed but did not.
  end
end
```

This seems to have something to do with #389. Removing the line loading one of DVTAnalyticsClient, DVTAnalytics or SourceKit seems to make the problem disappear...

**Comment by segiddins** [comment]
This is a known issue with Xcode 8 breaking `Dir.chdir`

**Comment by alloy** [comment]
See https://github.com/CocoaPods/Xcodeproj/pull/404

**Comment by neonichu** [comment]
Has been fixed by #407
